free-programming-books-et.md

```diff
@@ -30,15 +33,29 @@
 ### JavaScript
 
 * [JavaScript](http://puhang.tpt.edu.ee/raamatud/JavaScript_konspekt.pdf) - Jüri Puhang (PDF)
+* [Javascript](http://lepo.it.da.ut.ee/~epuman/javascript.html) - Ella Puman, Tartu ÜLikooli õppematerjal
 
 
+##### Angular
+
+* [Angular](http://www.cs.tlu.ee/teemad/get_file.php?id=400) - Tallinna Ülikooli õppematerjal
+
+#### React Native
+
+* [React Native](https://digikogu.taltech.ee/en/Download/fa57d958-b6eb-4bf9-a39c-ba843f3d4c30) - Roger Kerse
```

Comment:
Don't invent titles! Use the title given by the resource. title is "REACT NATIVE RAAMISTIKU KASUTAMINE MOBIILIRAKENDUSE UNISPOTTER NÄITEL" (but no allcaps titles!)

Reply:
Fixed.